prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -511,6 +534,8 @@ ssize_t rxr_pkt_post_read_to_copy_data(struct rxr_ep *rxr_ep,
 		return err;
 	}
 
+	peer = rxr_ep_get_peer(rxr_ep, rxr_ep->rdm_self_addr);
```

Comment:
I missed this as well in the earlier review. In addition to increasing this counter we need to check for `-FI_EAGAIN` from the read call and queue this read request when the TX queue is full.

Reply:
Updated the PR and added a 3rd commit to implement queue of read request.